### ISSUE TITLE
Compatibility of the library with OS X 10.7.5

### ISSUE THREAD

**Issue by lendenmc** [deanishe/alfred-workflow#27]
I've been trying to make the Python-written workflow [alfred-pocket](https://github.com/fniephaus/alfred-pocket) work on a Mac running Lion (10.7.5). However I am not raising an issue about the workflow per se, but about the `alfred-workflow` library it depends on. A solution could help for other Python-written Alfred workflows that use this library and need to run on Lion. Needless to say, the machine I use cannot be upgraded beyond Lion. 
So in my specific case, this workflow wasn't supposed to work on Lion but I wanted to know why it should not. In the workflow's debug window, I found out that the error I came across was caused by the following lines (1893-1894) in the `workflow.py` file of the library

``` python
retcode, password = self._call_security('find-generic-password',
                                        service, account, '-w')
```

In Lion, the `-w` option actually doesn't exist for the `security find-generic-password` command.
So this means that _the `alfred-workflow` library is not fully compatible with OS X versions prior to 10.8_.
In my case, I just replaced `-w` by the `-g` option in `workflow.py`, i.e.

``` python
retcode, password = self._call_security('find-generic-password',
                                         service, account, '-g')
```

This makes the error disappear.
However, the workflow isn't still working (although it works the first time, it crashes later on), as other authentication errors show up. It is close to work though. 
 **I would like to know whether other tweaks in the source code of the library might be needed to make _the library_ (not the workflow) fully compatible with Lion** (so that I can move on to the workflow specific code in a second step).
Thanks

**Comment by deanishe** [comment]
Should all be fixed in 0551765

**Comment by ChrisDeveloper** [comment]
Awesome job @deanishe, thanks a lot. It fixed the whole issue. I was able to test it on Lion with the above-mentioned [alfred-workflow] (https://github.com/fniephaus/alfred-pocket) and now it makes it at least compatible with 10.7 and, I guess, 10.6 as well. Will likely make many others workflows work on 10.6 and above. Sweet !
